test/cbmc/proofs/run-cbmc-proofs.py

```diff
@@ -334,20 +334,20 @@ async def configure_proof_dirs( # pylint: disable=too-many-arguments
         queue.task_done()
 
 
-def add_tool_version_job():
-    cmd = [
-        "litani", "add-job",
-        "--command", "cbmc-starter-kit-print-tool-versions .",
-        "--description", "printing out tool versions",
-        "--phony-outputs", str(uuid.uuid4()),
-        "--pipeline-name", "print_tool_versions",
-        "--ci-stage", "report",
-        "--tags", "front-page-text",
-    ]
-    proc = subprocess.run(cmd)
-    if proc.returncode:
-        logging.critical("Could not add tool version printing job")
-        sys.exit(1)
+#def add_tool_version_job():
```

Comment:
Why is this code commented out? Should it be removed?

Reply:
Hi Archit, I'm planning on re-enabling this for all repositories (not all of them currently have it), but in a separate PR. Is that OK?

Reply:
That is fine; in that case would be best for this PR to remove the commented out code, and can add it back in the other PR when ready

Reply:
OK, done. Thank you